recipe-client-addon/bin/make-xpi.sh

```diff
@@ -23,7 +23,7 @@ rm "${DEST}/install.rdf.in"
 cp "${BASE_DIR}/install.rdf" "${DEST}"
 rm "${DEST}/jar.mn"
 cp "${BASE_DIR}/chrome.manifest" "${DEST}"
-rm "${DEST}/test" -r
```

Comment:
Yep, I think OSX uses BSD. Does this change conflict with the GNU implementation? Should I reverse it?